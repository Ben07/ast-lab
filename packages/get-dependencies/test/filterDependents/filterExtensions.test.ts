// @ts-ignore
import { filterDependents } from './utils';

describe('filterDependents()', () => {
  test('filter extensions', async () => {
    const ret = await filterDependents(
      'imports', 
      ['importCss.js'], 
      { './importCss.js': ['default'] },
      {
        extensions: ['.js']
      }
    )
    expect(ret).toMatchSnapshot();
  });
});
