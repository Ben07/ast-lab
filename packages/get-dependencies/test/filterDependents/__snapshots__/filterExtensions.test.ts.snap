// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`filterDependents() filter extensions 1`] = `
Array [
  "/imports/importCss.js",
]
`;
